src/app.jsx

```diff
@@ -13,6 +14,18 @@ export default function App() {
         <Settings />
       </div>
       <div className="app__dashboard">
+        <SearchBar />
```

Comment:
i think this search bar is supposed to be deleted. and the one below buttons is supposed to be move up